### DISCUSSION TITLE
Welcome to aci-exporter Discussions!

### DISCUSSION THREAD

**Discussion by thenodon** [opsdis/aci-exporter#46] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wilberh** [comment]
Hi - besides Goland and Docker installation options, any plans for a Helm Charts installation option as well?

**Comment by thenodon** [comment, marked as answer]
Hi @wilberh. You should checkout https://github.com/datacenter/aci-monitoring-stack created by @camrossi. That project include a complete stack for ACI fabric monitoring based on helm including the aci-exporter.
I would recommend you wait to after the release of aci-exporter 0.8.0 that is just a couple of days away and will be the base version in https://github.com/datacenter/aci-monitoring-stack

**Reply by thenodon** [reply]
Version 0.8.0 is now released https://github.com/opsdis/aci-exporter/releases/tag/v0.8.0
